packages/core/src/generateKernel.js

```diff
@@ -41,16 +41,16 @@ function generateKernel (opts = {}) {
   if (opts.hasOwnProperty('scuttleGlobalThis')) {
```

Comment:
If we want to check [nullish `opts`](https://github.com/LavaMoat/LavaMoat/pull/462/files#diff-33db6acd3ecc68c97a8e15b1c2e45441dd24f84b6eb3a6b8d64d86ca3c6ae516R44): Here.

```suggestion
  if (opts?.hasOwnProperty('scuttleGlobalThis')) {
```